packages/client/components/NewMeeting.tsx

```diff
@@ -118,12 +116,38 @@ const createMeetingOrder = ({standups}: {standups: boolean}) => {
   return meetingOrder
 }
 
+const query = graphql`
+  query NewMeetingQuery {
+    viewer {
+      ...NewMeetingExistingMeetings_viewer
+      featureFlags {
+        standups
+      }
+      teams {
+        ...NewMeetingTeamPicker_selectedTeam
+        ...NewMeetingSettings_selectedTeam
+        ...NewMeetingTeamPicker_teams
+        ...NewMeetingActions_team
+        id
+        lastMeetingType
+        name
+        tier
+      }
+    }
+  }
+`
+
 const NewMeeting = (props: Props) => {
-  const {teamId, viewer, retry} = props
+  const {teamId} = props
+  const {queryRef} = props
+  const data = usePreloadedQuery<NewMeetingQuery>(query, queryRef, {
+    UNSTABLE_renderPolicy: 'full'
+  })
+  const {viewer} = data
   const {teams, featureFlags} = viewer
   const newMeetingOrder = useMemo(() => createMeetingOrder(featureFlags), [featureFlags])
 
-  useStoreQueryRetry(retry)
+  // useStoreQueryRetry(retry)
```

Comment:
actually, I'm not sure if the `retry` is still needed here, since the `refetch` is already set inside the `useQueryLoaderNow`